metagenomics_pipeline_qPCRnorm.ipynb

```diff
@@ -3649,7 +3649,9 @@
     "         'BarcodesAreRC': 'False',\n",
     "         'ForwardAdapter': 'GATCGGAAGAGCACACGTCTGAACTCCAGTCAC',\n",
     "         'ReverseAdapter': 'GATCGGAAGAGCGTCGTGTAGGGAAAGGAGTGT',\n",
-    "         'HumanFiltering': 'True'\n",
+    "         'HumanFiltering': 'True',\n",
+    "         'library_construction_protocol': '',\n",
+    "         'experiment_design_description': '',\n",
```

Comment:
Same as with the amplicon notebook, we need something that will definitely get updated by the user and that won't silently make its way to Qiita.